tests/graphql/test-expr-http.graphql

```diff
@@ -0,0 +1,27 @@
+#> server-sdl
+schema @server @upstream(baseURL: "http://abc.com") {
+  query: Query
+}
+
+type Query {
+  foo(id: Int!): User @expr(body: {http: {baseURL: "http://jsonplacheholder.typicode.com", path: "/users/{{args.id}}"}})
+}
```

Comment:
You can combine all the successful tests into one file `test-expr-success.graphql`.

Add new tests for invalid `@expr` usage with proper validation messages.